frontend/ui/widget/doublespinwidget.lua

```diff
@@ -102,20 +102,28 @@ function DoubleSpinWidget:update()
     }
     local right_widget = NumberPickerWidget:new{
         show_parent = self,
-        value = self.right_value,
+        value = numberpicker_right_value or self.right_value,
         value_min = self.right_min,
         value_max = self.right_max,
         value_step = self.right_step,
         value_hold_step = self.right_hold_step,
         precision = self.right_precision,
         wrap = self.right_wrap,
     }
+    left_widget.picker_updated_callback = function(value)
+        self:update(value, right_widget:getValue())
+    end
+    right_widget.picker_updated_callback = function(value)
+        self:update(left_widget:getValue(), value)
+    end
+    
```

Comment:
```
Checking frontend/ui/widget/doublespinwidget.lua  1 warning
    frontend/ui/widget/doublespinwidget.lua:119:1: line contains only whitespace
Total: 1 warning / 0 errors in 474 files
```
To be fixed on your next PR (no need to make a dedicated PR, it is minor, and you'll make a real PR soon enough :)